comfino.php

```diff
@@ -621,6 +579,41 @@ private function getFormFields()
         return $fields;
     }
 
+    private function addOrderStates()
+    {
+        $languages = Language::getLanguages(false);
+
+        foreach (OrdersList::ADD_ORDER_STATUSES as $state => $name) {
+            $newState = Configuration::get($state);
+
+            if (!$newState || empty($newState) || !Validate::isInt($newState) ||
```

Comment:
Myślę, ze można uprościć
```suggestion
            if (empty($newState) || !Validate::isInt($newState) ||
```

Nie potrafię sobie wyobrazić sytuacji, gdzie inaczej by się to zachowywało

> empty() is essentially the concise equivalent to !isset($var) || $var == false. 
https://www.php.net/manual/en/function.empty.php